app/hooks/use-price-conversion.ts

```diff
@@ -9,21 +9,22 @@ import {
 } from "@app/types/amounts"
 import * as React from "react"
 import { useMemo } from "react"
-import { useRealtimePriceWrapper } from "./use-realtime-price"
 
 export const SATS_PER_BTC = 100000000
 
 export const usePriceConversion = () => {
-  const { data } = useRealtimePriceWrapper({ fetchPolicy: "cache-first" })
+  const { data } = useRealtimePriceQuery({ fetchPolicy: "cache-first" })
```

Comment:
Why does `useRealtimePriceQuery` not have to be skipped here but skipped in the `useDisplayCurrencyHook`

Reply:
that's an oversight.